src/controllers/FacilityController.ts

```diff
@@ -62,7 +61,7 @@ export class FacilityController {
       const { numSpaces } = req.body;
 
       const repository = new SpaceAvailabilityRepository(facilityId, spaceId);
-      await repository.createDefaultAvailability(Number(numSpaces));
+      await repository.setAvailabilityDefault({ numSpaces: numSpaces });
```

Comment:
This can leads to error because `numSpaces` can be `string`

Reply:
Fixed.